_layouts/default-grid.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
Could we potentially use an if statement to include the grid based on whether or not the page being rendered is the homepage? (At least, I think what I'm seeing is that the homepage is the only one that doesn't use a grid.) Then we could combine the two.